README.md

```diff
@@ -38,6 +38,9 @@ in [application.conf](src/main/resources/application.conf)
 
 ## Spinning down Heroku prototypes
 
+> **Note**
+> With the introduction of the prototype auto-publish functionality we now automatically delete prototypes from heroku after 90 days unless they are listed as a protected prototype in the B&D API. So the current spin down approach is somewhat superseded by that, and in the future we may explore additional automated idling of running prototypes not being actively used. If you want to add a prototype to the list of protected prototypes, ask in #team-plat-ui on slack.
+
```

Comment:
unrelated to this change

Reply:
@oscarduignan Can we add somewhere in the `README` that we have currently pinned the version?